pyproject.toml

```
@@ -0,0 +1,50 @@
+[project]
+name = "q2-virsorter2"
+authors = [
+    { name = "Christos Matzoros", email = "christosmatzoros@gmail.com" }
+]
+description = "QIIME 2 plugin for identification and analysis of viral sequences."
+readme = {file = "README.md", content-type = "text/markdown"}
+license = {file = "LICENSE"}
+dynamic = ["version"]
+
+[project.urls]
+Homepage = "https://github.com/bokulich-lab/q2-virsorter2"
+Repository = "https://github.com/bokulich-lab/q2-virsorter2"
+
+[project.entry-points.'qiime2.plugins']
+"q2-virsorter2" = "q2_virsorter2.plugin_setup:plugin"
+
+[build-system]
+requires = [
+    "setuptools",
+    "versioningit",
+    "wheel"
+]
+build-backend = "setuptools.build_meta"
+
+[tool.versioningit.vcs]
+method = "git-archive"
+describe-subst = "$Format:%(describe)$"
+default-tag = "0.0.1"
+
+[tool.versioningit.next-version]
+method = "minor"
+
+[tool.versioningit.format]
+distance = "{base_version}+{distance}.{vcs}{rev}"
+dirty = "{base_version}+{distance}.{vcs}{rev}.dirty"
+distance-dirty = "{base_version}+{distance}.{vcs}{rev}.dirty"
+
+[tool.versioningit.write]
+file = "q2-virsorter2/_version.py"
```

```suggestion
file = "q2_virsorter2/_version.py"
```
There was a typo in my original template for this filepath (that I used to open all of the PRs for the setup ->pyptoml migration) - you'll see a few other PRs that I've opened with the same change (as well as metagenome -> moshpit distro target change). Sorry for missing this on my first go-around!